www/js/logging.js

```diff
@@ -0,0 +1,119 @@
+define([
```

Comment:
Following on our earlier conversations about function names:

I think we want functions with names centered around verbs (e.g. "setSession", "getInstance") rather than nouns (e.g. "logger").  Right now the calls in the other files look like "this.logging.logger(...)"  But I think it'd be easier to understand if it was written "this.logger.log(...)".  So I suggest the following name changes:

Methods:
- logger --> log or logEvent (maybe logEvent, to make it look different from a console.log call)
- eventLogger --> logUserEvent, logUIEvent, logUIAction, or logUserAction (whichever name is most accurate)
- clientLog --> logClientEvent

File/variable:
- logging.js --> logger.js
- logging --> logger

Suggestions welcome.